projects/public-search/src/app/patron-profile/patron-profile.service.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PatronProfileService {
+
+  /** Tabs event */
+  private _tabsEvent = new Subject<ITabEvent>();
+
+  /** Cancel event */
+  private _cancelRequestEvent = new Subject<string>();
+
+  /** Get tab event observable */
+  get tabsEvent$(): Subject<ITabEvent> {
+    return this._tabsEvent;
+  }
```

Comment:
Why private attribute and a getter returning it without any change protection ?

Reply:
Missing `asObservable` on variable